### PR TITLE
Create grc path

### PR DESCRIPTION
**Pull request by grcpath**
GRC PATH Logo

### CODE REVIEW

**Comment by vanbroup** [comment]
@grcpath thanks for this pull request, but please note that the pull request is empty.

The image in the comment above doesn’t seem to be an original vector but be traced from a jpg/png, can you provide an original vector such as .ai or .svg?

File: assets/images/members/grc path [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (vanbroup) [inline assets/images/members/grc path:1]: No image here, it’s empty

**Review by vanbroup — CHANGES_REQUESTED** [review]

**Comment by grcpath** [comment]
Dear PaulThanks alot for your replyI have attached the SVG file. I will try to add it again and pull request. Best, AliSent from my Galaxy
-------- Original message --------From: Paul van Brouwershaven ***@***.***> Date: 2/9/24  10:49 PM  (GMT+03:00) To: "pkic/pkic.org" ***@***.***> Cc: grcpath ***@***.***>, Mention ***@***.***> Subject: Re: [pkic/pkic.org] Create grc path (PR #111) 
@grcpath thanks for this pull request, but please note that the pull request is empty.
The image in the comment above doesn’t seem to be an original vector but be traced from a jpg/png, can you provide an original vector such as .ai or .svg?

**Comment by grcpath** [comment]
GRC Path Logo (1)

**Comment by grcpath** [comment]
Hi 
I had upload ne svg file

File: assets/images/members/grc-path/grc-path.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
...qrqZq0D8AAAAAAAA8Oqqt+le/alcTfKA6AQAA///s3bEJgEAQRNFflqE9WpGR1mAtBnJwNVzie7CwBexmAyNMAms9c45q/N82gyX73FXiAAAAAAAA8FejuuauzhkguarXNcBi1QcAAP//7N2xCYBQEETBF9qQgZGtixjZj2Aowu9BkBnYCi7YZOGMSeA7b/EdI6+pmqt1ZBlvcgAAAAAAAOCPrjEe2aq9OqvbpeFj1QMAAP//7NyxCYBQEETB14hdCAa2Z2diINiFjYjwi1CYgavggk2WVSaB73iDcR+3jeWSuVqrZSyXTP4FAAAAAADAT92jPHJWR3VZHoEPqh4AAAD//+zasQkAIAwAwdRu4P7LOIkjWFmIEMERFO5A0gop0ryYBN4185i264c1o5LzdmxS7BAAAAAAAIDHjIxFTjyyZ7ck+EBELAAAAP//7NoxCYAAAETRjxXs4G5yo7hbwMkKIoKTAQThPfgJbj1nEviXvVqebkM1VfOr0a4AAAAAAAB85KjWV1t1GgB+qLoAAAD//+zaIQqAUAwG4L96CLvh3dvrCCKY3yWsIiy8bFO+D8bytrCweSaBb3sW8FmxDpXM9VzSkixDnswbAAAAAACAl666Sx2V9yRbkq6h8CNJbgAAAP//7NyxCcJgFIXRb4yUKQQh42eWIIRYCFZuIYG/yAx6DtwBHrzycpVJ4De9R9bLdeeKyTyKJUt1r24jkz8AAAAAAABg+FRH9az26lFt1cvaCPyB6gsAAP//7NwxCoAwEATA7fUNFoL/B/+jhW/QXg6uEJ+gM7BcHRJIs4kyCfxHXex7Z32tengUS+YkS5KpfzipOTonAAAAAAAAn3H2w+SjU6WRrWflstXwY0luAAAA///s3CEOhEAQBMDWp1Cc4P8P4hEk3CkUGtPrQGOqkklG75oRnRYmAdKDYO3cmRosGeGSJcmc5NuZ227y8ZoAAAAAAACvOZPsSf5tF/l13xoaGe32hy8CHiW5AAAA///s2ksKwjAYReEzEa2gtvtflvuwpQW1OFACt5CZhc7K+eBCXgRCMstvMYmkNYbk/mftOUUlXdJWWfoX4AqcktI+ZLwBjtV+N29HkiRJkiRJkiTt2Ah8c7wZeAET8MncOxkzXv5r+urvZskjRSRPH4ukzYAfAAAA//8DAMIUfktPNQxNAAAAAElFTkSuQmCC"/>
```
Comment (vanbroup) [inline assets/images/members/grc-path/grc-path.svg:2]: This is not a vector image but a vector with an embedded png.

 `xlink:href="data:img/png;base64,`

Comment (vanbroup) [reply assets/images/members/grc-path/grc-path.svg:2]: @grcpath any updates on this?

Comment (svenska-primekey) [reply assets/images/members/grc-path/grc-path.svg:2]: @grcpath Have you had a chance to get an SVN logo without the embedded png?

**Comment by svenska-primekey** [comment]
Closing, no update from GRC Path to provide a SVG logo.

**Pull request closed without merging**